src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String EMPTY_STRING = "";
+    private static final String VALID_LOGIN = "Alexandr";
+    private static final String INVALID_LOGIN = "Alex";
+    private static final String VALID_PASSWORD = "Alex12345";
+    private static final String INVALID_PASSWORD = "12345";
+    private static final int VALID_AGE = 20;
+    private static final int INVALID_AGE = 17;

```

Comment:
I consider constants in tests a bad pattern

simply put - it makes your test methods less readable, you always need to scroll to find a corresponding value
apart from that, it makes your test method depend on a single constant, which gets even worse, if you use that constant in several places

while constants are ok for production code, they're not ok for test code :)

I recommend reading this [article](https://medium.com/@erik.sacre/on-tdd-literals-versus-constants-in-tests-8ac992c7873f)